src/main/java/com/datastax/oss/protocol/internal/request/Startup.java

```diff
@@ -23,31 +23,49 @@
 import java.util.Map;
 
 public class Startup extends Message {
-  private static final String CQL_VERSION_KEY = "CQL_VERSION";
-  private static final String COMPRESSION_KEY = "COMPRESSION";
+  public static final String CQL_VERSION_KEY = "CQL_VERSION";
+  public static final String COMPRESSION_KEY = "COMPRESSION";
 
   private static final String CQL_VERSION = "3.0.0";
 
-  public final Map<String, String> options;
+  private final Map<String, String> options;
```

Comment:
We can keep a public final field here since the map is immutable.
The protocol classes are very simple and low-level, so they generally expose fields directly instead of getters.